man/memkind.3

```diff
@@ -524,6 +530,38 @@ section if it is not.
 .BR MEMKIND_PMEM_MIN_SIZE
 The minimum size which allows to limit the file-backed memory partition.
 .sp
+.B "STATISTICS:"
+.br
+The functions described in this section define a way to get specific memory allocation statistics.
+.PP
+.BR memkind_update_cached_stats ()
+is used to force an update of cached dynamic allocator statistics.
+Statistics are not updated real-time by memkind library and this method allows to force update it on demand.
```

Comment:
`and this method allows to force its update.`

Reply:
Done